djangocms_link/models.py

```diff
@@ -60,6 +65,35 @@ class AbstractLink(CMSPlugin):
     target = models.CharField(_('target'), blank=True, max_length=100,
                               choices=TARGET_CHOICES)
 
+    # These attributes are already managed by other fields
+    EXCLUDED_KEYS = [
+        'href',
+        'target',
+    ]
+    attributes = AttributesField(
+        _('link tag attributes'), excluded_keys=EXCLUDED_KEYS,
+        help_text=_('Optional. Link HTML tag attributes'),
+    )
+
+    @cached_property
+    def _excluded_keys(self):
+        return [key.lower() for key in self.EXCLUDED_KEYS]
+
+    def attributes_str(self):
+        """
+        Emits self.attributes as a String for adding to an HTML element.
+        """
+        # Notice that we are explicitly ignoring keys in `EXCLUDED_KEYS` here
+        # too. The `excluded_keys` attribute on the AttributesField above is
+        # used to prevent *new* keys from being created that are in
+        # `EXCLUDED_KEYS`. The check here is to also omit *existing* keys that
+        # may already exist in the database.
+        attrs = []
+        for key, val in self.attributes.items():
+            if key.lower() not in self._excluded_keys:
+                attrs.append('{key}="{value}"'.format(key=key, value=conditional_escape(val)))
```

Comment:
django has a utility that does this afaik.

Also shouldn't this logic be in the core of djangocms-field-attributes?
Otherwise I can see us doing this in every package that implements it.